### DISCUSSION TITLE
Changing the need to fully press L and R Button in games

### DISCUSSION THREAD

**Discussion by Gitomata** [emukidid/swiss-gc#638] [Ideas]
Hi emukidid.

Wonderful work here. My suggested idea is whether you can implement a hack that makes the game recognise partially pressing the analogue L and R shoulder buttons as a fully pressing down to the 'click' for each L and R shoulder buttons.

Example, in Prince of Persia Sands of Time it's a bit clunky that the wall run move requires me to fully press the R shoulder button until it clicks and a partial press doesn't register as anything.  Can you change it where partially pressing the shoulder buttons is registered as a full click?  I'm sadly not a coder but I saw SWISS is able to invert the x and y axis on the c-stick to create invert camera options for games that don't support it so maybe this might be similar? 

I realise this option won't work for games that use the range of motion for the L and R shoulder buttons so this is more so for games that require having to press down all the way for a move to be registered. Being able to have a partially pressed L and R button register for these games, in my humble opinion, would make the controls a little more nimble.   

Thanks for reading this.

**Comment by emukidid** [comment]
@Extrems has just implemented this in record time it seems.

**Reply by Gitomata** [reply]
Awesome! Love your and @Extrems passion and dedication on SWISS and GameCube.
